Makefile

```diff
@@ -186,6 +187,11 @@ ol3cesium: .build-artefacts/ol3-cesium
 	git fetch -a; \
 	git checkout $(OL3_CESIUM_VERSION); \
 	git submodule update --recursive --init --force; \
+	cd cesium; \
+	git remote | grep c2c || git remote add c2c git://github.com/camptocamp/cesium; \
+	git fetch c2c; \
+	git checkout $(CESIUM_VERSION); \
+	cd ..; \
```

Comment:
What's in the c2c patch? Will we rely on c2c version of cesium in the future?

Discussion point next wee.